packages/api-docs/src/const-enum-overrides.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * ☝️ NOTE:
+ * Types in this file provide const enums replacement that will compile with isolatedModules
+ */
+
+import type { ApiItemKind as _ApiItemKind } from '@microsoft/api-extractor-model';
```

Comment:
unfortunately those packages heavily rely on const enums, so to be able to have `isolatedModules` this is the only workaround